apisix/plugins/jwt-auth.lua

```diff
@@ -35,6 +38,10 @@ local schema = {
             enum = {"HS256", "HS384", "HS512", "RS256", "ES256"}
         },
         exp = {type = "integer", minimum = 1},
+        base64_secret = {
```

Comment:
we need to check `secret` value if `base64_secret` is true.

It should be a valid base64 encoded string.

Reply:
I think we can merge fisrt, then add schema check, Chinesse doc and more test cases in another PR.

Reply:
agree +1